client/components/navbar.jsx

```diff
@@ -21,20 +21,26 @@ export default class Navbar extends React.Component {
   render() {
     return (
       <>
-        <nav className="relative flex flex-wrap items-center justify-between px-2 py-3 bg-zinc-200 border border-gray-200">
-          <div className="w-96 md:w-[800px] m-auto container px-4 flex flex-wrap items-center justify-between">
-            <div className="w-full relative flex justify-between md:w-auto md:static md:block md:justify-start">
-              <a className="pl-2 styled-font text-3xl leading-relaxed inline-block mr-4 py-2 whitespace-nowrap"
+        <nav className="relative flex flex-wrap items-center justify-between
+          px-2 py-3 bg-zinc-200 border border-gray-200">
+          <div className="w-96 md:w-[800px] m-auto container px-4 flex flex-wrap
+            items-center justify-between">
+            <div className="w-full relative flex justify-between md:w-auto md:static
+              md:block md:justify-start">
+              <a className="text-sky-600 pl-2 styled-font text-3xl leading-relaxed
+                inline-block mr-4 py-2 whitespace-nowrap"
                 href="#">
                 Foodies
               </a>
               <button className="cursor-pointer leading-none md:hidden"
                 onClick={this.handleNavbarChange}>
                 {!this.state.navbarOpen
-                  ? <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8" fill="none" viewBox="0 0 24 24" stroke="currentColor" strokeWidth={2}>
+                  ? <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8" fill="none"
+                      viewBox="0 0 24 24" stroke="currentColor" strokeWidth={2}>
```

Comment:
Please separate out all of your SVGs into their own components with readable names.